gcc/jit/dummy-frontend.cc

```diff
@@ -1372,7 +1403,8 @@ jit_langhook_global_bindings_p (void)
 static tree
 jit_langhook_pushdecl (tree decl ATTRIBUTE_UNUSED)
 {
-  gcc_unreachable ();
+  /* Do nothing to avoid crashing on some targets.  */
+  return NULL_TREE;
```

Comment:
What triggers the call to jit_langhook_pushdecl?

Reply:
Some backends like the Darwin and the Aarch64 ones.